src/dotnet/commands/dotnet-compile/CompilerCommandApp.cs

```diff
@@ -118,52 +93,13 @@ public int Execute(OnExecute execute, string[] args)
                 RuntimeValue = _runtimeOption.Value();
                 VersionSuffixValue = _versionSuffixOption.Value();
 
-                IsNativeValue = _nativeOption.HasValue();
-                ArchValue = _archOption.Value();
-                IlcArgsValue = _ilcArgsOption.HasValue() ? _ilcArgsOption.Values : Enumerable.Empty<string>();
-                IlcPathValue = _ilcPathOption.Value();
-                IlcSdkPathValue = _ilcSdkPathOption.Value();
-                AppDepSdkPathValue = _appDepSdkPathOption.Value();
-                IsCppModeValue = _cppModeOption.HasValue();
-                CppCompilerFlagsValue = _cppCompilerFlagsOption.Value();
-
-                // Set defaults based on the environment
-                var settings = ProjectReaderSettings.ReadFromEnvironment();
-
-                if (!string.IsNullOrEmpty(VersionSuffixValue))
-                {
-                    settings.VersionSuffix = VersionSuffixValue;
-                }
-
-                // Load the project file and construct all the targets
-                var targets = ProjectContext.CreateContextForEachFramework(ProjectPathValue, settings).ToList();
-
-                if (targets.Count == 0)
-                {
-                    // Project is missing 'frameworks' section
-                    Reporter.Error.WriteLine("Project does not have any frameworks listed in the 'frameworks' section.");
-                    return 1;
-                }
-
-                // Filter the targets down based on the inputs
+                var files = new ProjectGlobbingResolver().Resolve(_projectArgument.Values);
+                IEnumerable<NuGetFramework> frameworks = null;
                 if (_frameworkOption.HasValue())
                 {
-                    var fx = NuGetFramework.Parse(_frameworkOption.Value());
-                    targets = targets.Where(t => fx.Equals(t.TargetFramework)).ToList();
-
-                    if (targets.Count == 0)
-                    {
-                        // We filtered everything out
-                        Reporter.Error.WriteLine($"Project does not support framework: {fx.DotNetFrameworkName}.");
-                        return 1;
-                    }
-
-                    Debug.Assert(targets.Count == 1);
+                    frameworks = new [] { NuGetFramework.Parse(_frameworkOption.Value()) };
```

Comment:
FrameworkOption is single value, yet the var is plural. Should the frameworks option also be multiple value? Also, should users be able to specify frameworks for each project, or is there just a single set of frameworks and all projects must support the entire set?

In extension to this, how do the other command args interact with multiple projects, like output, buildbasepath, etc?